src/gap.h

```diff
@@ -16,30 +16,6 @@
 #include "system.h"
 
 
-/****************************************************************************
-**
-*V  Last  . . . . . . . . . . . . . . . . . . . . . . global variable  'last'
-**
-**  'Last',  'Last2', and 'Last3'  are the  global variables 'last', 'last2',
-**  and  'last3', which are automatically  assigned  the result values in the
-**  main read-eval-print loop.
-*/
-extern UInt Last;
-
-
-/****************************************************************************
-**
-*V  Last2 . . . . . . . . . . . . . . . . . . . . . . global variable 'last2'
-*/
-extern UInt Last2;
-
-
-/****************************************************************************
-**
-*V  Last3 . . . . . . . . . . . . . . . . . . . . . . global variable 'last3'
-*/
-extern UInt Last3;
```

Comment:
@wucas in reply to your question: yes, you guessed right; removing the above lines from `gap.h` in a sense is the central purpose of this PR. Reducing the "surface" of the kernel makes it easier to reason about, easier to use it as a library, easier to make future changes. Of course this PR is just a tiny contribution to this overall goal.